web-server/routes/calendar.test.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+const loadDigest = require('../../digest')
+const cheerio = require('cheerio')
+
+describe('GET /calendar route', function() {
+
+  whenNotLoggedIn(function(){
+    describe('GET /calendar, not logged in', function() {
```

Comment:
dont put "not logged in" in the describe message here. There is already a parent desribe that says that. just say `'GET /calendar'`